### DISCUSSION TITLE
Chatbot message cap

### DISCUSSION THREAD

**Discussion by Daniel-DDV** [FlowiseAI/Flowise#1593] [Q&A]
Hi,
I set a message limit for one of my chatbots.
However, it does not seem to respect it on production. 
Is this perhaps due to caching?
Thanks in advance,
Daniel

EDIT: I am hosting on Render and created Environment Variable NUMBER_OF_PROXIES and set it to 1 according to these instructions: https://docs.flowiseai.com/configuration/rate-limit

Still doesn't respect the rate limit.

EDIT 2:
I verified with {{hosted_url}}/api/v1/ip and my own IP doesn't match. So I increased NUMBER_OF_PROXIES by 1 and repeated this to no avail. The IP address changes, but randomly and does not match my own.

**Comment by mwmdev** [comment]
I have the same issue with Flowise 2.1.1 hosted on docker on a VPS.
